cypress/integration/shared/clusterConfiguration.ts

```diff
@@ -78,6 +83,41 @@ export const waitForHostTablePopulation = (
   });
 };
 
+export const waitForHostsSubnet = (cy) => {
```

Comment:
Please note we moved from plain JavaScript to TypeScript.
At the build type, static type-analysis is done. To make it possible, we need to specify types whenever they can not be deduced from context.

```suggestion
export const waitForHostsSubnet = (cy: Cypress.cy) => {
```

Reply:
Done.